packages/rpc-methods/src/restricted/manageAccounts.test.ts

```diff
@@ -104,7 +104,7 @@ describe('manageAccountsImplementation', () => {
         params: {},
       }),
     ).rejects.toThrow(
-      'At path: method -- Expected a string, but received: undefined',
+      'Expected the value to satisfy a union of `object | object`, but received: [object Object]',
```

Comment:
The error message isn't great since the type is a union now :(

Reply:
We have an open issue for better error messaging, which will probably fix this error too. This is partially implemented in the CLI already, and results in much easier to understand errors:

https://github.com/MetaMask/snaps/blob/4b7ca517a366a063f1eaad3fd18fb4d1f3afb105/packages/snaps-utils/src/structs.test.ts#L122-L130

Reply:
Good to know, thanks!